service-commands/scripts/create-instance.sh

```diff
@@ -101,6 +101,11 @@ case "$provider" in
 		;;
 	gcp)
 		gcloud compute instances create $name $(gcp_image_to_flags $image) --boot-disk-size $disk_size --machine-type $machine_type 2> /dev/null
+		gcloud compute disks create $name-solana-disk \
```

Comment:
we shouldn't need to increase the size here. we should be able to use our existing disk size which is already very generous